osf/models/preprint_provider.py

```diff
@@ -30,6 +30,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_facebook = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_instagram = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
+    blog = models.URLField(null=True, blank=True)
```

Comment:
`URLField` has a [default max_length of 200](https://docs.djangoproject.com/en/1.9/ref/models/fields/#urlfield), figured I'd not change it.